src/Microsoft.DotNet.Configurer/LocalizableStrings.resx

```diff
@@ -126,13 +126,14 @@ Telemetry
 --------------
 The .NET Core tools collect usage data in order to improve your experience. The data is anonymous and does not include command-line arguments. The data is collected by Microsoft and shared with the community.
 You can opt out of telemetry by setting a DOTNET_CLI_TELEMETRY_OPTOUT environment variable to 1 using your favorite shell.
-You can read more about .NET Core tools telemetry @ https://aka.ms/dotnet-cli-telemetry.
-
-Configuring...
+You can read more about .NET Core tools telemetry @ https://aka.ms/dotnet-cli-telemetry.</value>
```

Comment:
One thing though: loc'ed version will show old message. We can force everything back to English in interim by renaming original key if that's a concern. I don't think it matters as long as everything settles down for 2.0 RTM, though.